libs/utils/android/workloads/uibench.py

```diff
@@ -85,7 +85,7 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         self.out_dir = out_dir
         self.collect = collect
 
-        # Press Back button to be sure we run the video from the start
+        Screen.unlock(self._target)
         System.menu(self._target)
```

Comment:
Re: First comment, there's some weirdness where if I don't do it in unlock() then the screen doesn't unlock. I didn't get too much time last time around to fix that, but I will investigate more today.

Re: Second comment, I normally don't push changes without testing them so I didn't change what I wasn't using ;-) although, now that you mentioned it, I will try to include it in this patch with some amount of testing